app/views/buyers/service_contracts/new.html.erb

```diff
@@ -11,11 +11,11 @@
       <p><%= h @service.name %></p>
     </li>
 
-    <%= form.input :plan, :collection => @service_plans, :include_blank => !@service_contract.plan_id? %>
+    <!--  It will prompt only if there is no default service plan selected -->
+    <%= form.input :plan, collection: @service_plans, prompt: !@service_contract.plan_id? %>
```

Comment:
I have a default service plan selected and the pop up is showing
![Screen Shot 2019-08-02 at 16 26 22](https://user-images.githubusercontent.com/11672286/62377030-47e26880-b542-11e9-80a9-1971ed159684.png)

Reply:
@josemigallas do you have the default plan set? or only have a plan named `default`? 

please check this page and screenshot http://provider-admin.example.com.lvh.me:3000/apiconfig/services/{{ ID }}/application_plans

to remove the `please select`, you need to have a `default plan` marked here

Reply:
It is checked, yes. The name is just to reassure it hehe

Reply:
I see no "Please select" with default plan
_____
<img width="749" alt="Screenshot 2019-08-02 at 17 02 58" src="https://user-images.githubusercontent.com/13486237/62379579-84649300-b547-11e9-9fa0-e7eb4d9d34ae.png">
_____

It's true there is a weird scroll bar. But this issue is about disallowing blank plan option, then I'd say it's OK. And we should create a new task to fix the scrolling issue which is happening not only here but in all the modals